docs/data/sdks/typescript-browser/index.md

```diff
@@ -394,3 +394,29 @@ amplitude.init(API_KEY, OPTIONAL_USER_ID, {
 #### Disable cookies
 
 You can opt out using cookies by setting `disableCookies` to `true` so that the SDK will use `LocalStorage` instead. `LocalStorage` is a great alternative, but because access to `LocalStorage` is restricted by subdomain, you can't track anonymous users across subdomains of your product (for example: `www.amplitude.com` vs `analytics.amplitude.com`).
+
+--8<-- "includes/sdk-device-id/lifecycle-header.md"
+
+1. Device id in configuration on initialization
```

Comment:
Can you align the `id`?
I have seen `id`, `iD`, and `ID`.

Reply:
Both `device ID` and `device id` are widely used in our dev doc center. Maybe we need a better way to unify them. 
Keep it as how it is now.